config.py.example

```diff
@@ -0,0 +1,122 @@
+########################################
+# APP-SPECIFIC CONFIGURATION VARIABLES #
+########################################
+# The following variables should or must be defined in your
+# own application. Create a config.py file, import that config_common
+# file as:
+#
+#   from config_hub import *
+#
+# then define the following variables to fit your needs. You can also override any
+# any other variables in this file as required. Variables defined as ValueError() exceptions
+# *must* be defined
+#
+
+import logging
+
+from biothings.utils.loggers import setup_default_log
+from biothings.utils.configuration import ConfigurationError, ConfigurationDefault, ConfigurationValue
```

Comment:
In this `config.py.example`, we probably should not use `ConfigurationError, ConfigurationDefault, ConfigurationValue`, but using an example value (e.g. `localhost` for `DATA_SRC_SERVER`). That's easier for users to modify.

Reply:
It's updated